omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_8/ConceptSearchHandler1_8.java

```diff
@@ -73,9 +76,28 @@ public PageableResult search(RequestContext context) throws ResponseException {
 		String name = context.getParameter("name");
 		String conceptClass = context.getParameter("class");
 		String searchType = context.getParameter("searchType");
+		String conceptUuids = context.getParameter("uuids");
 		
 		List<Concept> concepts = new ArrayList<Concept>();
 		
+		if (StringUtils.isNotBlank(conceptUuids)) {
+			Concept concept = null;
+			for (String conceptUuid : conceptUuids.split(",")) {
+				if (!conceptUuid.trim().equalsIgnoreCase("")) {
```

Comment:
`StringUtils.isNotBlank()` (from Apache Commons Lang)is a much more efficient and correct way to do this (basically, `trim()` will create a new String).

Reply:
> `StringUtils.isNotBlank()` (from Apache Commons Lang)is a much more efficient and correct way to do this (basically, `trim()` will create a new String).

I didn't know that trim() would rather create a new String.